### ISSUE TITLE
Qt doesn't support dark mode on windowsvista

### ISSUE THREAD

**Issue by Desoroxxx** [PrismLauncher/PrismLauncher#653]
### Operating System

Windows

### Version of Prism Launcher

Prism Launcher 6.1 (Always happened not matter the version)

### Version of Qt

At 6.4.0

### Description of bug

Prism just does not follow system theme no matter what.
It's the only app on my system to do that.

![image](https://user-images.githubusercontent.com/82710983/208621708-cc5e63b6-33e7-4d55-b871-838a96df7a8d.png)

![image](https://user-images.githubusercontent.com/82710983/208621719-44d8d442-2c76-44bf-b87f-8d2be582d73d.png)

### Steps to reproduce

Be on Windows, launch prism with any theme it will not follow, change theme while prism is launch it will not follow.

### Suspected cause

Not sure as I am using Windows 11 and I know some changes have been made to theme maybe prism isn't listening at the correct place.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by DioEgizio** [comment]
https://bugreports.qt.io/browse/QTBUG-72028

TL;DR the windows API doesn't give dark palettes and there's no way to have a "native" dark mode on windows

**Comment by Desoroxxx** [comment]
But you can get if the user uses light or dark theme with that info you could switch between user defined light and dark themes?

**Comment by DioEgizio** [comment]
Yes we can do that

**Comment by Desoroxxx** [comment]
With this build: https://github.com/PrismLauncher/PrismLauncher/actions/runs/8528576742

The original issue of Prism Launcher not being able to dynamically switch between dark and light theme is fixed.

But this is still not possible:
> switch between user defined light and dark themes

Also dynamically switching isn't really perfect:
![image](https://github.com/PrismLauncher/PrismLauncher/assets/82710983/f2f966ef-332d-477d-b27f-7ddb8ebe70f7)
(This doesn't go away unless I restart)

Should this be keep in this state, or should I close this and open two new issues?

**Comment by DioEgizio** [comment]
> (This doesn't go away unless I restart)

That looks like an upstream Qt issue

> switch between user defined light and dark them

I assume you mean being able to make themes that can change whether you're on dark mode or light?

**Comment by Desoroxxx** [comment]
> I assume you mean being able to make themes that can change whether you're on dark mode or light?

I use catpuccin themes, I wished I could do like the system theme does but with catpuccin, defining a light and dark theme in the Prism settings and Prism switching between those depending on my system theme, just like the System theme except with user defined themes.

**Comment by Trial97** [comment]
We no longer support windows vista

**Comment by TheKodeToad** [comment]
This is just the theme for windows vista and upwards which includes Windows 10

**Comment by Trial97** [comment]
is either this issue or https://github.com/PrismLauncher/PrismLauncher/issues/129 which one do you want to keep open?

**Comment by Desoroxxx** [comment]
Should I open a new issue to `switch between user defined light and dark themes`

**Comment by Trial97** [comment]
Do you mean https://github.com/PrismLauncher/PrismLauncher/issues/362?

**Comment by Desoroxxx** [comment]
Yes actually
